### PR TITLE
Gdb 10937 update explain functionality with the new model

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
The explain response models have been updated.

## Why
The backend exposes new fields in the response, providing an opportunity for further UX improvements.

## How
The UI models have been updated to reflect the changes.

## Additional work
- The actions remain visible until the explain response is expanded;
- The raw query is hidden if it is identical to the query;
- The raw query text ellipsis when the query is too long;
- Hide the "Copy SPARQL Query" button if the query in queryMethod is not SPARQL;
- Change the name of the query label depending on the query type: "JSON" or "SPARQL."

### CODE REVIEW

File: test-cypress/stubs/ttyg/ttyg-stubs.js [hunk 1]
```diff
@@ -178,4 +178,78 @@ export class TTYGStubs extends Stubs {
...
+        cy.intercept({
+            method: 'POST',
+            url: 'rest/chat/conversations/explain'
+        }, (req) => {
+            const requestBody = req.body;
+
+            const explainResponse = {
+                conversationId: requestBody.conversationId,
+                answerId: requestBody.answerId,
+                queryMethods: [
```
Comment (svilenvelikov) [inline test-cypress/stubs/ttyg/ttyg-stubs.js:192]: You are often using such smarter stubs but I don't like them. They are harder to be maintained and updated.

Comment (svilenvelikov) [reply test-cypress/stubs/ttyg/ttyg-stubs.js:192]: why don't you just use fixtures?

Comment (boyan-tonchev) [reply test-cypress/stubs/ttyg/ttyg-stubs.js:192]: but they are closer to how the real system works.

Comment (boyan-tonchev) [reply test-cypress/stubs/ttyg/ttyg-stubs.js]: Done

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**